Issue1〜3の修正

Coffeeplantation opened the pull request:
Issue1〜3の解消

Coffeeplantation committed:
issue1

Coffeeplantation committed:
issue2

Coffeeplantation committed:
issue3

TakashiKozuma commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,7 +18,7 @@ def plan_params
   end
 
-  def getWeek
+  def get_Week
```
※5,21行目
```rb
get_Week
```
こちらはメソッド名ですので「頭文字大文字のキャメルケース」から「小文字のスネークケース」に変更する必要があります。
現状、キャメルケースとスネークケースが混ざった書き方になってしまっていますので、
改めてカリキュラムを見直して「小文字のスネークケース」に修正しましょう！

TakashiKozuma commented on app/views/calendars/index.html.erb:
```diff
@@ -23,5 +23,4 @@
-  </div>
-</div>
\ No newline at end of file
+  </div>
```
現状だと、divの閉じタグだけ削除してしまっている状態です。
対応する開始タグも削除しましょう！
classなどを何も指定していない部分に注目してみてください！

TakashiKozuma reviewed (commented):
レビュー依頼ありがとうございます！
コメントを致しましたので、ご確認お願いいたします。

修正後はコミット・プッシュのし忘れがないかご確認の上、再度フォームからレビュー依頼をお願い致します。

Coffeeplantation committed:
issue2の修正

下記修正
----------
※5,21行目

get_Week
こちらはメソッド名ですので「頭文字大文字のキャメルケース」から「小文字のスネークケース」に変更する必要があります。
現状、キャメルケースとスネークケースが混ざった書き方になってしまっていますので、
改めてカリキュラムを見直して「小文字のスネークケース」に修正しましょう！

Coffeeplantation committed:
issue3の修正

16行目を削除

div-reviewer reviewed (commented):
Coffeeplantation様、学習お疲れ様です！
お忙しい中issue1-3の再レビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

完成まで後もうひと踏ん張りですね。最後まで頑張っていきましょう！
修正点をコメントさせて頂きましたので、ご確認をお願い致します。

```ruby
【次回プルリクエスト提出する際の注意点】
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②googleフォームの送信をお願いいたします。
```

ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください！
どうぞよろしくおねがいします。

Coffeeplantation committed:
issue3訂正

15行目を復元

Coffeeplantation committed:
issue3修正３回目

div-reviewer reviewed (commented):
Coffeeplantation様、学習お疲れ様です！
夜遅くまで学習お疲れさまです。引き続きissueのレビュー依頼ありがとうございます。プルリクエスト確認させていただきました。

issue1,2についてはクリアできているので、issue3について再チャレンジ頑張りましょう。
修正点をコメントさせて頂きましたので、ご確認をお願い致します。

```ruby
【次回プルリクエスト提出する際の注意点】
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②googleフォームの送信をお願いいたします。
```

ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください！
どうぞよろしくおねがいします。

Coffeeplantation committed:
issue3

復元、及び、修正

zonodance commented on app/views/calendars/index.html.erb:
```diff
@@ -7,21 +7,20 @@
   </div>
-</div>
\ No newline at end of file
+  </div>
```
現時点で、まだ不要なdivタグが残っております。
もし途中からの修正が難しければ、下記を参考に再度実装に取り組んでみましょう。
```ruby
<div>
  <%= form_with model: @plan, url: calendars_path, class: 'form' do |f| %>
    <%= f.label :日付を選択, class: 'lavel' %>
    <%= f.date_field :date, class: 'date-select' %>
    <%= f.label :予定を入力, class: 'lavel' %>
    <%= f.text_field :plan, class: 'text-input' %>
    <%= f.submit '保存' %>
  <% end %>
</div>
<div>
  <div class='calendar'>
    <% @week_days.each do |day| %>
      <div class='item'>
        <div class='date'>
          <% day[:month] %>/<% day[:date] %>
        </div>
        <ul class='content'>
        <% if day[:plans].length != 0 %>
          <% day[:plans].each do |plan| %>
            <li class='plan-list'>・<%= plan %></li>
          <% end %>
        <% end %>
        </ul>
      </div>
    <% end %>
  </div>
</div>
```

zonodance reviewed (commented):
レビュー依頼ありがとうございます！
コメント内容を確認の上再度実装に取り組んでみましょう。

Coffeeplantation committed:
issue3

10行目の<div>を削除

TAKADA-yuma commented on app/views/calendars/index.html.erb:
```diff
@@ -7,21 +7,21 @@
     <%= f.submit '保存' %>
```
こちらのファイルに余分な記述があるようです。
そちらを修正しましょう！

詳しくは[カリキュラム](https://master.tech-camp.in/v2/curriculums/4681) のIssue３をご確認ください。

TAKADA-yuma reviewed (commented):
レビュー依頼ありがとうございます。プルリクエスト確認させていただきました。


修正点を追加でコメントさせて頂きましたので、ご確認をお願い致します。

```ruby
【次回プルリクエスト提出する際の注意点】
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②既に頂いている挙動確認の画像や動画が修正範囲だった場合、最新の画像や動画の提出をお願い致します。
③googleフォームの送信をお願いいたします。
```
ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください！
どうぞよろしくおねがいします。

Coffeeplantation committed:
issue3

訂正

TAKADA-yuma commented:
- 削除するdivタグは２セットになります。
- divタグは<div> </div>のセットで行いましょう。
　片方のみ削除してしまうとエラーやビュー崩れが起きる可能性があります。

もし途中からの修正が難しければ、下記を参考に再度実装に取り組んでみましょう。
```
<div>
  <%= form_with model: @plan, url: calendars_path, class: 'form' do |f| %>
    <%= f.label :日付を選択, class: 'lavel' %>
    <%= f.date_field :date, class: 'date-select' %>
    <%= f.label :予定を入力, class: 'lavel' %>
    <%= f.text_field :plan, class: 'text-input' %>
    <%= f.submit '保存' %>
  <% end %>
</div>
<div>
  <div class='calendar'>
    <% @week_days.each do |day| %>
      <div class='item'>
        <div class='date'>
          <% day[:month] %>/<% day[:date] %>
        </div>
        <ul class='content'>
        <% if day[:plans].length != 0 %>
          <% day[:plans].each do |plan| %>
            <li class='plan-list'>・<%= plan %></li>
          <% end %>
        <% end %>
        </ul>
      </div>
    <% end %>
  </div>
</div>
```

Coffeeplantation committed:
issue3

修正

zonodance commented on app/views/calendars/index.html.erb:
```diff
@@ -7,21 +7,18 @@
-</div>
\ No newline at end of file
+        </div>
+    <% end %>
```
再度こちらのビューファイルを修正しましょう。
①一度、元の記述に戻していただく
（上記は必須ではありませんが、一度最初の記述に戻していただくほうがスムーズに進む可能性があります。）
元の記述
```ruby
<div>
  <%= form_with model: @plan, url: calendars_path, class: 'form' do |f| %>
    <%= f.label :日付を選択, class: 'lavel' %>
    <%= f.date_field :date, class: 'date-select' %>
    <%= f.label :予定を入力, class: 'lavel' %>
    <%= f.text_field :plan, class: 'text-input' %>
    <%= f.submit '保存' %>
  <% end %>
</div>
<div>
  <div class='calendar'>
    <% @week_days.each do |day| %>
      <div class='item'>
        <div class='date'>
          <% day[:month] %>/<% day[:date] %>
        </div>
        <ul class='content'>
        <% if day[:plans].length != 0 %>
          <% day[:plans].each do |plan| %>
            <li class='plan-list'>・<%= plan %></li>
          <% end %>
        <% end %>
        </ul>
      </div>
    <% end %>
  </div>
</div>
```
②カリキュラムのヒントを元に、不要なdivタグがないか整理する
ご不明点等がございましたら、コネクトメンターまでご相談ください。

zonodance reviewed (commented):
レビュー依頼ありがとうございます！
コメント内容を確認の上、再度実装に取り組んでみましょう。

Coffeeplantation committed:
issue3

復元

Coffeeplantation committed:
issue3

訂正

TAKADA-yuma reviewed (commented):
レビュー依頼ありがとうございます！
おめでとうございます！LGTMです！
Issue１〜３の解消、お疲れ様でした！
Issue３に関しては特に何度もトライしていただいて、ありがとうございました！

この後の流れの確認です。
①Git上でマージ
②ローカルのmasterブランチへ切り替え
③Fetch Originしプル

そのあと、second trainingにブランチを切り替えてIssueの解消に取り組みましょう。

引き続き、実装頑張っていきましょう。
応援しております！

Coffeeplantation commented:
お世話になっております。
何度も申し訳ありませんでした。
やっと頂いたヒントから<div></div>の意味がわかりました。
次のissue4~6もよろしくお願い致します。

市川

> 2023/01/18 21:29、TAKADA-yuma ***@***.***>のメール:
> 
> 
> @TAKADA-yuma commented on this pull request.
> 
> レビュー依頼ありがとうございます！
> おめでとうございます！LGTMです！
> Issue１〜３の解消、お疲れ様でした！
> Issue３に関しては特に何度もトライしていただいて、ありがとうございました！
> 
> この後の流れの確認です。
> ①Git上でマージ
> ②ローカルのmasterブランチへ切り替え
> ③Fetch Originしプル
> 
> そのあと、second trainingにブランチを切り替えてIssueの解消に取り組みましょう。
> 
> 引き続き、実装頑張っていきましょう。
> 応援しております！
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/Coffeeplantation/training_curriculum2/pull/8#pullrequestreview-1253385046>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A4GLIM7225FYMCAL2PYX7QTWS7O2LANCNFSM6AAAAAAT5Y2DKA>.
> You are receiving this because you authored the thread.
>
